staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -50,6 +50,7 @@ type WebhookAuthorizer struct {
 	authorizedTTL       time.Duration
 	unauthorizedTTL     time.Duration
 	initialBackoff      time.Duration
+	failOpen            bool
```

Comment:
failOpen is misleading... it doesn't allow on error, it defers. maybe `decisionOnError authorizer.Decision` instead?